packages/browserid-verifier/tests/lib/verifier.js

```diff
@@ -136,10 +137,14 @@ Verifier.prototype.stop = function (cb) {
   if (!this.process || !this._url) {
     throw new Error('verifier not running');
   }
-  this.process.kill('SIGINT');
   this.process.on('exit', function (code) {
     cb(!code ? null : 'non-zero exit code: ' + code);
   });
+  const pid = this.process.pid;
+  // Really really kill it. This shouldn't be necesary, but sometimes when
```

Comment:
Do we know what's stopping the process from ending? Are there open handles? If so is addressing them worth a follow up ticket?